### ISSUE TITLE
Renamed APK no longer found after update from 0.8.6 to 1.0.0

### ISSUE THREAD

**Issue by markuspfeiffer** [KeepSafe/dexcount-gradle-plugin#291]
I've just updated the plugin from 0.8.6 to 1.0.0. Since then, the build fails with the following message:

```
* What went wrong:
A problem was found with the configuration of task ':<app>:countNormalDebugDexMethods'.
> File '/Users/<user>/git/<app>-android/<app>/build/outputs/apk/normal/debug/<app>-normal-debug.apk' specified for property 'inputFileProperty' does not exist.
```

We use the following construct in our Gradle file to rename the APK:
```
android {
    applicationVariants.all { variant ->
        variant.outputs.all {
            outputFileName = "<app>-${variant.flavorName}-${variant.buildType.name}-${variant.versionName}.apk"
        }
    }
}
```

This works fine in 0.8.6, but seems to no longer be compatible with 1.0.0. I've gone back to 0.8.6 and builds work as expected again.

**Comment by benjamin-bader** [comment]
Thanks for the heads-up; I'll take a look!

**Comment by benjamin-bader** [comment]
Also, please let me know
- what Gradle version you're using
- what Android Gradle Plugin version you're using

**Comment by markuspfeiffer** [comment]
Hi, thanks for the quick feedback :)

I'm on Android Studio 3.5.2 with `com.android.tools.build:gradle:3.5.2`, Gradle (wrapper) 5.4.1. `compileSdkVersion = 28` and `buildToolsVersion = '28.0.3'`

Let me know if you need anything else. I'm afraid I can't share the project directly, since its for work.

**Comment by benjamin-bader** [comment]
Great, thanks - those versions (along with your snippet showing how you're renaming the APK) should be sufficient for a repro.  Will let you know.

**Comment by benjamin-bader** [comment]
Repro'd and fixed.  I'll have a point release up soon, will comment here when it's live.

**Comment by benjamin-bader** [comment]
I just pushed version 1.0.1 to Maven Central, and it should be available in the next 15-60 minutes.

**Comment by markuspfeiffer** [comment]
I've tested with 1.0.1, unfortunately this results in a different error (during Gradle sync). I've tried the suggestions (re-download etc.), but that doesn't help.

```
ERROR: Unable to find method 'org.gradle.api.file.RegularFileProperty.fileProvider(Lorg/gradle/api/provider/Provider;)Lorg/gradle/api/file/RegularFileProperty;'.
```

Are you using a different Gradle version?

**Comment by benjamin-bader** [comment]
Well that's no good :/
Looks like I goofed.  This method was introduced in Gradle 6, not in 5.1 like I had thought.  I'll have to do another point release.  Thanks for your patience!

**Comment by benjamin-bader** [comment]
I've got a PR open, but it's 2:00 AM and I'd like to sleep rather than wait for CI and do the Maven dance.  I'll push the release tomorrow.  Thanks again.

**Comment by markuspfeiffer** [comment]
Hey, it's fine, no rush :)

**Comment by benjamin-bader** [comment]
I released version 1.0.2 with the above fix; it should be in Maven Central within the hour.

**Comment by markuspfeiffer** [comment]
Confirmed working on my end. Thanks a lot for your quick response :)
